src/JestConfigEditor.ts

```diff
@@ -3,6 +3,7 @@ import { Config, ConfigEditor } from 'stryker-api/config';
 import JestConfigLoader from './configLoaders/JestConfigLoader';
 import DefaultJestConfigLoader from './configLoaders/DefaultJestConfigLoader';
 import ReactScriptsJestConfigLoader from './configLoaders/ReactScriptsJestConfigLoader';
+import ReactScriptsTsJestConfigLoader from './configLoaders/ReactScriptsTsJestConfigLoader';
```

Comment:
`TS` should be uppercase right? Like `DB` and `MQ`.

Reply:
I don't know, and to be honest, I don't really care either. If you prefer `TS`, `TS` it will be ;-).